### PR TITLE
Add --url option back

### PR DESCRIPTION
**Pull request by dustymabe**
Resolves: rhbz#1250115

### CODE REVIEW

File: plugins/download.py [hunk 1]
```diff
@@ -56,6 +57,9 @@ def set_argparser(parser):
                             help=_('download path, default is current dir'))
         parser.add_argument('--resolve', action='store_true',
                             help=_('resolve and download needed dependencies'))
+        parser.add_argument('--url', action='store_true',
+                            help=_('just list http(s) urls where the rpms can '
```
Comment (ignatenkobrain) [inline plugins/download.py]: I think it's better to `s/just/print/`

File: plugins/download.py [hunk 2]
```diff
@@ -128,9 +146,34 @@ def _download_debuginfo(self, pkg_specs):
...
+            # have a mirrorlist we can then just choose one of the mirrors
+            # and merge the mirror url with the relative location of the
+            # package on the mirror.
+            for pkg in pkgs:
+                h = pkg.repo._get_handle()
+                h.fetchmirrors = True  # sets LRO_FETCHMIRRORS
+                h.destdir = tmpdir     # store tmp files in tmpdir
+                h.perform()
+                for mirror in h.mirrors:
+                    if mirror.startswith('https'):
```
Comment (ignatenkobrain) [inline plugins/download.py]: hm, what about ftp or http?

Comment (dustymabe) [inline plugins/download.py]: oops, I originally had this as http, not https. but I decided to try out https and forgot to change it back. Will fix it to http.

My thoughts about ftp and rsync is that they are generally less useful than the http(s) ones. almost everything can use http(s). The alternative to this is that we give some option for the user to specify what protocol they want. I think that is overcomplicating it though. What do you think?

**Comment by dustymabe** [comment]
@ignatenkobrain - tests should pass now

**Comment by ignatenkobrain** [comment]
Looks good to me now, can you add it into documentation and squash commits?

**Comment by dustymabe** [comment]
@ignatenkobrain - added docs and got it down to the smallest number of commits that I'd like to keep.

**Comment by dustymabe** [comment]
@ignatenkobrain anything left to do on this?

**Comment by ignatenkobrain** [comment]
@dnf-bot r+

**Comment by j-mracek** [comment]
@dnf-bot retry

**Comment by j-mracek** [comment]
@dnf-bot retry

**Pull request closed without merging**